content/en/watchdog.md

```diff
@@ -16,62 +16,108 @@ further_reading:
   text: "Collect your traces"
 ---
 
-{{< vimeo 278057125 >}}
+<div class="alert alert-warning">
+Watchdog infrastructure stories are in private beta. <a href="/help">Contact Datadog support</a> to have them enabled for your organization.
```

Comment:
We will no longer be in private beta, once this goes out. We should probably remove this warning. cc @belenkil